docker-compose.yml

```diff
@@ -74,3 +84,32 @@ services:
       - redis
       - es
       - mc
+  chrome:
+    profiles:
+      - test
+    image: selenium/standalone-chrome
+    ports:
+      - "4444:4444"
+  test:
+    profiles:
+      - test
+    build:
+      context: .
+      dockerfile: ./config/docker/Dockerfile
+    environment:
+      - RAILS_ENV=test
+      - CHROME_URL=http://chrome:4444
+      - DOCKER=true
+      - CAPYBARA_PORT=5100
+      - OTWA_NO_START_REDIS=1
```

Comment:
So I should modify this config file to use `DOCKER` instead of `OTWA_NO_START_REDIS`? https://github.com/otwcode/otwarchive/blob/cbb45213e7ab83fcfd84a584e79dd7abcf7df720/config/initializers/gem-plugin_config/redis.rb#L8-L11

Reply:
Yeah.